test.sh

```diff
@@ -27,8 +27,8 @@ fi
 function setup_osbs() {
   # Optionally specify repo and branch for osbs-client to test changes
   # which depend on osbs-client patches not yet available in upstream master
-  OSBS_CLIENT_REPO=${OSBS_CLIENT_REPO:-https://github.com/containerbuildsystem/osbs-client}
-  OSBS_CLIENT_BRANCH=${OSBS_CLIENT_BRANCH:-master}
+  OSBS_CLIENT_REPO=${OSBS_CLIENT_REPO:-https://github.com/mkosiarc/osbs-client}
```

Comment:
:eyes:

Reply:
otherwise tests would fail, this will be removed before merging.

Reply:
just making note :)